config.go

```diff
@@ -0,0 +1,132 @@
+package kafkalock
+
+import (
+	"errors"
+	"os"
+	"path"
+	"time"
+
+	"github.com/Shopify/sarama"
+)
+
+// Dictates if/how KafkaLock should "validate" the topic configuration.
+type TopicPolicy uint8
+
+const (
+	// No validation will be done at all. KafkaLock will misbehave if the topic is not configured properly.
+	TopicPolicyNoValidation TopicPolicy = iota
+	// If the topic isn't configured as expected or doesn't exist at all then the KafkaLocker will return an error.
+	TopicPolicyValidateOnly
+	// If the topic isn't configured as expected then the KafkaLocker will return an error. If the topic doesn't exist
+	// at all then it will be created with the expected configuration. This is recommended (and default) configuration.
+	TopicPolicyCreate
+	// If the topic isn't configured as expected then it will be deleted and then re-created with the expected
+	// configuration (same if the topic doesn't exist at all). This isn't a recommended configuration as it will hide
+	// errors in the KafkaLocker configuration and will cause misbehaviour if multiple KafkaLocker instances try to
+	// acquire lock for the same topic but will use different configurations. Use this for manual "maintenance" only.
+	TopicPolicyDropAndCreate
+)
+
+// Struct holding the KafkaLocker configuration. Use NewConfig() to create one with defaults.
+type Config struct {
+	// Contains list of Kafka bootstrap servers (servers to which KafkaLock will connect to).
+	BootstrapServers []string
+	// Sarama's configuration struct - use it to tune timeouts to your liking. Leave nil to use the defaults.
+	KafkaConfig *sarama.Config
+	// Name of the topic you wan't to use for the lock. It serves as the lock identifier (the "thing" you are locking).
+	Topic string
+	// Maximum number of KafkaLocker instances that may hold the lock to certain topic at the same time.
+	// For exclusive lock use MaxHolders = 1 (default).
+	MaxHolders int
+	// Replication factor says on how many brokers we want the topic to be replicated on.
+	// Kafka's default is 3, so is ours.
```

Comment:
Default is `1`

Reply:
It is? Wow, I remembered somehow that it was 1...but I guess that was the recommended value? Nvm, changing the description

Reply:
Yup, for use in prod recommended is 3. Eg. linkedin uses 2. But default is 1 :P